app/controllers/verify/review_controller.rb

```diff
@@ -34,7 +34,7 @@ def create
     private
 
     def idv_profile_complete?
-      idv_session.resolution.try(:success?)
+      idv_session.profile_confirmation == true
```

Comment:
Do we want our code to reflect vendor language, or do we want it to be consistent for developers to understand? I was trying to make each step consistent, since to the user, it's just a step in the proofing process. Each "step" involves submitting a form and getting a pass or fail response. Maybe `*_confirmation` is not the right name either, but I think all steps should use the same name. Step 2 cannot begin unless Step 1 has been completed. `idv_profile_complete?` asks whether or not Step 1 has been completed. Since all the steps check whether they have been completed using the same method naming scheme, `idv_*_complete?`, I think it should be based on the same `idv_session` method since all it's asking is whether or not the step passed. It doesn't matter what the vendor calls this step IMO.